### PR TITLE
Update `mjml-react` to allow `letterSpacing` in buttons

### PR DESCRIPTION
**Pull request by nahtnam**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: See text below
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

---

`letter-spacing` support was added to `mjml` here: https://github.com/mjmlio/mjml/pull/1732

Looking at `mjml-react`, the button (https://github.com/wix-incubator/mjml-react/blob/261d996d9fb67492bd8168caaec50a11fd5f3a25/src/mjml-button.js) calls `handleMjmlProps` (https://github.com/wix-incubator/mjml-react/blob/261d996d9fb67492bd8168caaec50a11fd5f3a25/src/utils.js#L28) with all the props. That function just proxies all the props to `mjml` which leads me to believe no change is required in the `react-mjml` library. Furthermore, I tested locally and adding a `// @ts-ignore` and providing the `letterSpacing` prop works as expected

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## mjml-react ([<kbd>unpkg</kbd>](https://unpkg.com/browse/mjml-react@latest/))
was missing the following properties:
1. renderToJSON
1. renderToJSON2
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e7029fa8fb1a9c75c6b6674addb466d7ab81e5a1
</p>

**Comment by nahtnam** [comment]
Let me know if I need to update the tests and/or run that command

**Review by henrinormak — APPROVED** [review]

**Review by IanEdington — APPROVED** [review]

**Comment by nahtnam** [comment]
Hey! It said that I can't merge without a test (unless a DT maintainer approves) so I just wrote a simple test by using the prop and setting it to the default value. Let me know if that works for you!

**Review by IanEdington — APPROVED** [review]

**Pull request merged**
